service/history/api/respondactivitytaskfailed/api_test.go

```diff
@@ -538,7 +538,7 @@ func (s *workflowSuite) setupMutableState(uc UsecaseConfig, ai *persistencepb.Ac
 
 	currentMutableState.EXPECT().GetWorkflowType().Return(uc.wfType).AnyTimes()
 	if uc.expectRetryActivity {
-		currentMutableState.EXPECT().RecordLastActivityStarted(gomock.Any())
```

Comment:
Hmm... why are we not recording the last started time anymore?

Reply:
RecordLastActivityStarted is renamed to RecordLastActivityCompleteTime, because this is what it actually doing (LastAttemptCompleteTime).

Reply:
I'm probably missing some context here but we also sometimes record the start time in mutable state, where does this happen now (to save me from reading)?